### PR TITLE
Append .exe to crashreporter path so it is better located

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: src/gui/application.cpp [hunk 1]
```diff
@@ -246,8 +246,15 @@ Application::Application(int &argc, char **argv)
...
 
 #if defined(WITH_CRASHREPORTER)
-    if (ConfigFile().crashReporter())
-        _crashHandler.reset(new CrashReporter::Handler(QDir::tempPath(), true, CRASHREPORTER_EXECUTABLE));
+    if (ConfigFile().crashReporter()) {
+        auto reporter = QStringLiteral(CRASHREPORTER_EXECUTABLE);
+#ifdef Q_OS_WIN
+        if (reporter.endsWith(QLatin1String(".exe"))) {
+            reporter.append(QLatin1String(".exe"));
+        }
```
Comment (dschmidt) [inline src/gui/application.cpp:254]: I think this is the opposite of what you want to do  :trollface:

**Review by dschmidt — CHANGES_REQUESTED** [review]

File: src/gui/application.cpp (hunk 1 above)
Comment (dschmidt) [reply src/gui/application.cpp:254]: AFAICT it should never have .exe anyway tho...

Comment (TheOneRing) [reply src/gui/application.cpp:254]: your right the if is wrong 🙈 
But the crashreporter.exe is an exe so ....
if the full file name is passed it uses the exe close to the binary if available, else it falls back to path.

Comment (TheOneRing) [reply src/gui/application.cpp:254]: Ah yes we don't append exe yet, but ...

**Review by dschmidt — APPROVED** [review]

**Pull request merged**